### ISSUE TITLE
Update README to help onboard new users/participants

### ISSUE THREAD

**Issue by actuarialvoodoo** [kasaai/pc-pricing-tutorial#118]
It took me a little bit to figure out I needed to run make.R. Anyway we could borrow from devtools and have necessary files sourced via key command?

**Comment by kevinykuo** [comment]
Yeah agreed this is not beginner/newcomer friendly at all and we can do much better...
